client/app/components/group/pickupFeedback/pickupFeedback.spec.js

```diff
@@ -1,4 +1,6 @@
 import PickupFeedbackModule from "./pickupFeedback";
+import bagImg from "./bag.png";
```

Comment:
You could keep references to the images in the controller's constructor, then you don't need those. Would be _slightly_ cleaner to test.

```js
// constructor
   images: {
       bagImg, flourImg, // ...
   }

// test
expect($ctrl.isHigherImg($ctrl.flourImg)).to.be.false;
```